test/list_edit_matrix_test.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:collection/collection.dart';
+import 'package:json_patch/src/list_edit_matrix.dart';
+import 'package:test/test.dart';
+
+main() {
+  group('buildEditMatrix', () {
+    test('buildEditMatrix should handle empty lists', () {
```

Comment:
Would be nice to follow the same naming convention I mentioned earlier, i.e. "ListEditMatrix" group and ".buildEditMatrix handles empty list"

Reply:
👍